packages/contracts/contracts/optimistic-ethereum/OVM/execution/OVM_ExecutionManager.sol

```diff
@@ -1920,9 +2129,39 @@ contract OVM_ExecutionManager is iOVM_ExecutionManager, Lib_AddressResolver {
             (bool success, bytes memory returndata) = ovmCALL(
                 _transaction.gasLimit,
                 _transaction.entrypoint,
+                _value,
                 _transaction.data
             );
             return abi.encode(success, returndata);
         }
     }
+
+
+    /**
+     * Unreachable helper function for setting an account's balance so that eth_calls with value always work.  Only used by simulateMessage().
```

Comment:
Since this is meant to be unreachable I would recommend putting `require(msg.sender == address(0)`.

Reply:
done